webui/react/src/components/kit/Input.tsx

```diff
@@ -0,0 +1,25 @@
+import { Input as AntdInput, InputProps, InputRef } from 'antd';
+import React, { forwardRef, ForwardRefExoticComponent, RefAttributes } from 'react';
+
+import InputNumber from './InputNumber';
+import InputSearch from './InputSearch';
+
+const Input: Input = forwardRef<InputRef, InputProps>((props: InputProps, ref) => {
+  return <AntdInput ref={ref} {...props} />;
+}) as Input;
+
+type Input = ForwardRefExoticComponent<InputProps & RefAttributes<InputRef>> & {
```

Comment:
For Breadcrumb this approach makes sense to me since those are always used together, but for these I think we can just make these separate top-level components like `SearchBox` etc. The fact that they're backed by an <input> is an implementation detail.